pkgs/servers/seafile-server/default.nix

```diff
@@ -1,48 +1,60 @@
-{ stdenv
-, lib
-, fetchFromGitHub
-, pkg-config
-, python3
-, autoreconfHook
-, libuuid
-, sqlite
-, glib
-, libevent
-, libsearpc
-, openssl
-, fuse
-, libarchive
-, libjwt
-, curl
-, which
-, vala
-, cmake
-, oniguruma
-, nixosTests
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  python3,
+  autoreconfHook,
+  libuuid,
+  libmysqlclient,
+  sqlite,
+  glib,
+  libevent,
+  libsearpc,
+  openssl,
+  fuse,
+  libarchive,
+  libjwt,
+  curl,
+  which,
+  vala,
```

Comment:
A few of the function inputs appear to be unused, e.g libarchive, vala, oniguruma.

Reply:
`libarchive`, `vala`, and `oniguruma` are all needed dependencies, the build will not complete without them.
`openssl` and `curl` seem unused though.